draft-ietf-aipref-vocab.md

```diff
@@ -203,11 +203,144 @@ If arrangements, such as contracts exist that explicitly permit the use of that
+
+The format relies on the Dictionary type defined in {{Section 3.2 of !FIELDS=RFC9651}}.
+The dictionary keys correspond to usage categories
+and the dictionary values correspond to explicit preferences.
+
+For example, the following is a preference to allow AI training ({{ai}}),
+disallow generative AI training ({{genai}}), and
+and state no preference for other categories other than subsets of these categories:
+
+~~~
+ai=y, genai=n
+~~~
+
+
+## Usage Category Labels
+
+Each usage category in the vocabulary ({{vocab}}) is mapped to a short textual label.
+{{t-category-labels}} tabulates this mapping.
+
+| Category               | Label | Reference |
+|:-----------------------|:------|:----------|
+| Text and Data Mining   | tdm   | {{tdm}}   |
+| AI Training            | ai    | {{ai}}    |
+| Generative AI Training | genai | {{genai}} |
+{: #t-category-labels title="Mappings for Categories"}
+
+Any mapping for a new usage category can only use
+lowercase latin characters (a-z), digits (0-9), "_", "-", ".", or "*".
+These are encoded using the mappings in {{!ASCII=RFC0020}}.
+
+
+## Preference Labels
+
+The abstract model used has two options for preferences associated with each category:
+allow and disallow.
+These are mapped to single byte Tokens ({{Section 3.3.4 of !FIELDS}})
+of "y" and "n", respectively (without the quotes).
+
+
+## Text Encoding
+
+Structured Fields {{!FIELDS=RFC9651}} describes a byte-level encoding of information,
+not a text encoding.
+This makes this format suitable for inclusion in any protocol or format that carries bytes.
+
+Some formats are defined in terms of strings rather than bytes.
+These formats might need to decode the bytes of this format to obtain a string.
+As the syntax is limited to ASCII {{?ASCII=RFC0020}},
+an ASCII decoder or UTF-8 decoder {{?UTF8=RFC3629}} can be used.
+This results in the strings that this document uses.
+
+Processing (see {{processing}}) requires a sequence of bytes,
+so any format that uses strings needs to encode strings first.
+Again, this process can use ASCII or UTF-8.
+
+
+## Syntax Extensions
+
+There are two ways by which this syntax might be extended:
+the addition of new labels and the addition of parameters.
+
+New labels might be defined to correspond to new usage categories.
+{{vocab-extension}} addresses the considerations for defining new categories.
+New labels might also be defined for other types of extension
+that do not assign a preference to a usage category.
+In either case, when processing a parsed Dictionary to obtain preferences,
+any unknown labels MUST be ignored.
+
+The Dictionary syntax ({{Section 3.2 of !FIELDS}}) can associate parameters
+with each key-value pair.
+This document does not define any semantics for any parameters that might be included.
+When processing a parsed Dictionary to obtain preferences,
+any unknown parameters MUST be ignored.
+
+
+## Processing Algorithm {#processing}
```

Comment:
Why do we need this here?  Isn't this defined in RFC 9651?  If so, just point there.

Reply:
The processing in RFC 9651 gets you a dictionary, not our abstract model.